submissions/rafiatAminu/JavaScript/cond.js

```diff
@@ -0,0 +1,25 @@
+let a = 0;
+let b= 7;
+let c= 123;
+let d= (c-b);
```

Comment:
variable names should be descriptive enough. things like `a, b, c d` etc as a variable name is not proper

Reply:
I have worked on that as well. Kindly review @adaugochi Thanks